qemu/tests/cfg/pvpanic_basic.cfg

```diff
@@ -0,0 +1,20 @@
+- pvpanic_basic: install setup image_copy unattended_install.cdrom
+    virt_test_type = qemu
+    type = pvpanic
+    disable_shutdown = yes
+    monitor_type = qmp
+    event_check = "GUEST_PANICKED"
+    driver_name = "pvpanic"
+    timeout = 360
+    backup_image_before_testing = yes
+    restore_image_after_testing = yes
+    set_kdump_cmd = "chkconfig kdump off && systemctl stop kdump.service"
```

Comment:
Instead of 'chkconfig kdump off' it would be better to use 'systemctl disable kdump'

Reply:
Thanks for your advice, I will update it.